### PR TITLE
Csv geographica

### PR DESCRIPTION
**Pull request by ethervoid**
CSV task to get geographica data files

### CODE REVIEW

File: tasks/geographica/csv.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+                numercol=join_table['join_column']
+            ))
+        return '''SELECT {cols}
+                  FROM {geom} geom {joins}
+               '''.format(cols=' ,'.join(colnames), geom=join_data['geom']['table'],
+                          joins=' '.join(joins))
+
+    def _generate_csv_file(self, headers, measurements):
+        try:
+            with(open(self.output().path, 'w+')) as csvfile:
```
Comment (antoniocarlon) [inline tasks/geographica/csv.py]: I miss a `self.output().makedirs()` above this line

Comment (ethervoid) [reply tasks/geographica/csv.py]: good point, I was looking for something like this :)

File: tasks/geographica/csv.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+                    'block_group': 'us.census.tiger.block_group',
+                    'block': 'us.census.tiger.block'}
+
+
+class Measurements2CSV(Task):
+    geography = Parameter()
+    file_name = Parameter()
+
+    def __init__(self, *args, **kwargs):
+        super(Measurements2CSV, self).__init__(*args, **kwargs)
```
[the remark this answers is not in the document]
Comment (ethervoid) [reply tasks/geographica/csv.py]: I tend to add it when I inherit from plain task

**Comment by ethervoid** [comment]
@antoniocarlon changes done

**Pull request merged**